lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -605,6 +605,13 @@ class SemanticsObject {
       !hasAction(ui.SemanticsAction.tap) &&
       !hasFlag(ui.SemanticsFlag.isButton);
 
+  /// Whether this object is enabled.
+  ///
+  /// Relevant for objects such as controllers (ex: buttons, checkboxes)
+  bool get isEnabled =>
```

Comment:
SemanticsNodes can be in three distinct states (enabled, disabled, no opinion) depending on the state of the flags:

hasEnabledState set and isEnabled set: mark as enabled
hasEnabledState set and isEnabled not set: mark as disabled
hasEnabledState not set: no opinion whether it is enabled or disabled

This getter makes it seem like they are all either disabled or enabled.

Reply:
thanks a lot for the clarification! 

Since a boolean won't be enough I added an enum to carry the 3 different states.